gallery/__init__.py

```diff
@@ -1040,11 +1059,24 @@ def get_member_list(auth_dict: Optional[Dict[str, Any]] = None):
     return jsonify(ldap.get_members())
 
 
+@app.route("/member", methods = ["POST"])
+@auth.oidc_auth('default')
+@gallery_auth
+def member_mode(auth_dict: Optional[Dict[str, Any]] = None):
+    if session['userinfo'].get('member-lock'):
+        session['userinfo']['member-lock'] = not session['userinfo']['member-lock']
+    else:
+        session['userinfo']['member-lock'] = True
```

Comment:
```suggestion
    session['userinfo'].get('member-lock') = not session['userinfo']['member-lock']
```
this does the same thing the if did

Reply:
Not if it doesn't exist, this isn't pre-added to user info so we need to init it to the dict

Reply:
oh, maybe this to do the default?
```suggestion
    session['userinfo']['member-lock'] = not session['userinfo'].get('member-lock', False)
```